src/main/java/emissary/admin/PlaceStarter.java

```diff
@@ -82,7 +83,7 @@ public static IServiceProviderPlace createPlace(final String theLocation, final
      * @return the place that was found or created, or null if it can't be done
      */
     public static IServiceProviderPlace createPlace(final String theLocation, final Object[] constructorArgs, final String theClassStr) {
-        logger.debug("Ready to createPlace " + theLocation + " as " + theClassStr);
+        logger.debug("Ready to createPlace {} as {}", theLocation, theClassStr);
 
         final long t1 = System.currentTimeMillis();
 
```

Comment:
Since this PR is for Sonar and code cleanup should the TODO comment bellow on line 91 be addressed or removed? Another TODO on line 165

Reply:
I was going to handle the TODOs in a separate pr. Since it might change some functionality, maybe a good target for 8??